frog/imports/client/Wiki/index.js

```diff
@@ -347,7 +358,7 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
 
   changeTitle = (pageId, newPageTitle) => {
     changeWikiPageTitle(this.wikiDoc, pageId, newPageTitle);
-    const instanceId = this.props.match.params.instance;
+    const instanceId = this.props.match?.params.instance || this.props.embeddedPage.instance;
```

Comment:
Should be this.props.embeddedPage.instanceId

Reply:
Reverted to using this.props.embeddedPage.instance to match the naming convention of the Router